matlab/README.md

```diff
@@ -53,6 +53,10 @@ Supported `arrow.array.Array` types are included in the table below.
 | `datetime`        | `TimestampArray` |
 | `duration`        | `Time32Array`    |
 | `duration`        | `Time64Array`    |
+| `datetime`        | `Date32Array`    |
```

Comment:
Small nitpick - can we move the `Date32Array` and `Date64Array` rows to be just below the `TimestampArray` row?

Reply:
We also may want to indicate the default MATLAB -> Arrow type mapping for `datetime` and `duration`.

Reply:
> Small nitpick - can we move the `Date32Array` and `Date64Array` rows to be just below the `TimestampArray` row?

Yes, I can make that change.

Reply:
Sounds good!

Reply:
Marking this as resolved.